recipe/meta.yaml

```
@@ -27,7 +27,7 @@ build:
   # TODO: the specific on python does not work for now, we need to investigate why.
   # skip: true  # [libevent == "2.1.10" and py>39]
   # skip: true  # [libevent == "2.1.10" and py<39]
-  number: 3
+  number: 4
```

Ideally, the build number should be reset to 0 (rc3's should should have been reset to 0, I missed that).
```suggestion
  number: 0
```